ansible_builder/utils.py

```diff
@@ -1,8 +1,51 @@
 import subprocess
+import sys
+import os
+import yaml
 
+from . import constants
 
-def run_command(command):
+
+def run_command(command, capture_output=False):
     print('Running command:')
     print('  {0}'.format(' '.join(command)))
-    result = subprocess.run(command)
-    return bool(result.returncode == 0)
+
+    process = subprocess.Popen(command,
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.STDOUT)
+
+    output = []
+    for line in iter(process.stdout.readline, b''):
+        line = line.decode(sys.stdout.encoding)
+        if capture_output:
+            output.append(line.rstrip())
+        sys.stdout.write(line)
+
+    rc = process.poll()
+    return (rc, output)
+
+
+def introspect():
+    paths = []
+    path_root = os.path.join(constants.base_collections_path, 'ansible_collections')
```

Comment:
I really want to make this path configurable. That's lighting up all kinds of ideas in my head. This might get added as an option to _all_ commands. That's an interesting discussion to have, but doesn't need to be covered here in any way.